File: C/Calculator.c
#include <stdio.h>

int main() {
    int a, b;
    char op;
    
    printf("Enter first number: ");
    scanf("%d", &a);
    
    printf("Enter operator: ");
    scanf(" %c", &op);
    
    printf("Enter second number: ");
    scanf("%d", &b);
    
    switch (op) {
        case '+':
            printf("%d\n", a + b);
            break;
        case '-':
            printf("%d\n", a - b);
            break;
        case '*':
            printf("%d\n", a * b);
            break;
        case '/':
            if (b != 0)
                printf("%d\n", a / b);
            else
                printf("Error: Division by zero\n");
            break;
        default:
            printf("Invalid operator\n");
    }
    
    return 0;
}

File: PYTHON/Calculator.py
a = int(input("Enter first number: "))
op = input("Enter operator: ")
b = int(input("Enter second number: "))

if op == '+':
    print(a + b)
elif op == '-':
    print(a - b)
elif op == '*':
    print(a * b)
elif op == '/':
    if b != 0:
        print(a // b)  # Integer division like Java
    else:
        print("Error: Division by zero")
else:
    print("Invalid operator")
